plugins/grid-bookmark/src/GridBookmarkWidget/model.ts

```diff
@@ -115,49 +112,45 @@ export default function f(_pluginManager: PluginManager) {
         ]
       },
       get validAssemblies() {
-        return [
-          ...new Set(
-            this.assemblies.filter((assembly: string) =>
-              getSession(self).assemblyNames.includes(assembly),
-            ),
+        return new Set(
```

Comment:
changed validAssemblies to stay a Set, which hypothetically improves performance when you use validAssemblies.has(x) instead of validAssemblies.includes(x)